component.json

```diff
@@ -141,6 +141,70 @@
         }
       },
       "dynamicMetadata": true
+    },
+    "upsertSpreadsheetRow": {
+      "main": "./lib/actions/upsertSpreadsheetRow.js",
+      "title": "Create/Upsert/Update Spreadsheet Row",
+      "help": {
+        "description": "Action appends new row/column to spreadsheet or update it",
+        "link": "/components/gspreadsheet/actions#upsert-spreadsheet-row"
+      },
+      "fields": {
```

Comment:
Provide fields order to each field, like: `"order": 8,` (more number, higher field)